packages/components/ScreenContainer/index.tsx

```diff
@@ -126,7 +125,14 @@ export const ScreenContainer: React.FC<ScreenContainerProps> = ({
     <SafeAreaView style={{ width: "100%", flex: 1 }}>
       {/*FIXME: Too many containers levels*/}
 
-      <View style={styles.container}>
+      <View
+        style={{
+          flex: 1,
+          backgroundColor: "#000000",
```

Comment:
should we use `secondaryColor` instead of hardcoded one ?

Reply:
I think this PR is not the place to edit the styles